gradio/components/chatbot.py

```diff
@@ -194,6 +194,7 @@ def __init__(
         avatar_images: tuple[str | Path | None, str | Path | None] | None = None,
         sanitize_html: bool = True,
         render_markdown: bool = True,
+        feedback_options: Sequence[str] | None = ("Like", "Dislike"),
```

Comment:
In our docs page, Sequence gets automatically converted to List, which is confusing here since the default value is a tuple. Maybe this is better?
```suggestion
        feedback_options: list[str] | tuple[str] | None = ("Like", "Dislike"),
```